mkdocs.yml

```diff
@@ -145,6 +145,7 @@ theme:
   icon:
     edit: material/file-edit
     view: material/file-eye
+    repo: fontawesome/brands/github-alt
```

Comment:
use `fontawesome/brands/square-github` instead

Reply:
Doesn't fit with the other icons. I think the GitHub alt looks nice

Reply:
I think `fontawesome/brands/git-alt` looks good too.

Reply:
We had that before, but I don't think it matters what it's changed to. It's insignificant